src/components/organism/ToDoList/ToDoList.tsx

```diff
@@ -50,29 +92,40 @@ const ToDoList = ({selectionMode,UpdateCallback}: ToDoListProps) => {
             onTextClick={()=>{todoitemOnTextClick()}}
             onDelete={()=>{todoitemOnDelete()}}
             onUpdate={()=>{todoitemOnUpdate()}}/>
-        }
-
+        }        
   return (
-    <ToDoListContainer>
-        <ToDoListList>
-            {
-                uncompletedtodos.map((todo: todoprops) =>{ return todoitem(todo)})
-            }
-            { selectionMode? "":
-                completedtodos.map((todo: todoprops) =>{ return todoitem(todo)}) 
-            }
-        </ToDoListList>
-            <DeleteSelectedToDoDrawer 
-            onDeleteSelected={()=>{SDrawerOnDeleteSelected()}} 
-            onCompleteSelected={()=>{SDrawerOnCompleteSelected()}} 
-            onSelectAll={()=>{setSelectedToDos([...todos])}} 
-            hide={()=>setSelectedToDos([])} 
-            hidden={selectedToDos.length===0}/>
-            <DeleteToDoDrawer 
-            onYes={()=> {DrawerOnYes()}} 
-            hide={()=>{setDeletedrawer(true)}} 
-            hidden={deletedrawer}/>
-    </ToDoListContainer>
+    <Container>
+    {Filled?
+    <>
+        <ToDoListContainer className={Filled? "": "NoToDo"}>
+            <ToDoListList>
+                {sortedToDos.map((todo: todoprops) =>{ return todoitem(todo)})}
```

Comment:
just to confirm Sir, are perhaps referring to the fragment at the very outermost of this tertiary results true block? if yes then at the moment it is required there to return a single-parent component else react would throw an error.

Reply:
@Maurice63 ahh sorry. I'm referring to the `return todoitem(todo)` part. You can just immediately do `(todo: todoprops) => <ToDoItem ... />`

Reply:
for that part, it would also move the declaration of the additional helper functions that required the individual todo parameter of the .map inside the .map function and as it was in the return function of the list component I thought that it was better to move them in a separate helper function declared in the list component as to improve readability.

Reply:
I see. I guess it is better, but can you move the functions in `todoitem(todo)` to outside instead? My only concern is that the function is also iterating for every todo item in here. Just add a `todo` parameter for every function so it would know what object to make changes on.

Reply:
Okay, will do that, for the additional variables used in the helper functions I'll also add them as parameters to the todoitem declaration.